addons/html_editor/static/src/others/embedded_components/core/table_of_content/table_of_content.scss

```diff
@@ -8,16 +8,19 @@ $embedded-toc-bg--active: #1ba1e4;
     outline: 1px solid $o-gray-300;
     outline-offset: -1px;
     border-radius: 5px;
-    padding: 8px;
+    background-color: $o-gray-100;
+    overflow-y: auto;
```

Comment:
So we can have a scrollbar inside the table of content? Is it expected?

Reply:
Answered in the comment above.